cookbook/vectordb/cassandraDb.py

```diff
@@ -0,0 +1,36 @@
+from phi.agent import Agent
+from phi.knowledge.pdf import PDFUrlKnowledgeBase
+from phi.vectordb.cassandra.cassandra import CassandraDb
+import os
+
+try:
+    from cassandra.cluster import Cluster  # type: ignore
+except (ImportError, ModuleNotFoundError):
+    raise ImportError(
+        "Could not import cassandra-driver python package. " "Please install it with `pip install cassandra-driver`."
+    )
+from phi.embedder.mistral import MistralEmbedder
+from phi.model.mistral import MistralChat
+
+cluster = Cluster()
+session = cluster.connect("testkeyspace")
```

Comment:
Can we move this to the Cassandra class? and take the name as param

Reply:
Hey @manthanguptaa , The current implementation accepts the session as an argument to the CassandraVectorDb class to allow more flexibility

1- Users can configure and customize their Cassandra session (e.g., authentication, cluster settings, retries) outside the CassandraVectorDb class.
2- If the application already manages Cassandra sessions elsewhere, it avoids redundant session creation and allows sharing a single session across multiple components.

In the cookbook example, I have kept it simple by directly connecting to a local Cassandra cluster with no authentication. However, real-world production scenarios might require more complex connection setups.

Please let me know your thoughts on this approach.